pkg/services/ngalert/provisioning/notification_policies.go

```diff
@@ -86,6 +81,13 @@ func (nps *NotificationPolicyService) UpdatePolicyTree(ctx context.Context, orgI
 		return definitions.Route{}, "", err
 	}
 
+	revision.Config.AlertmanagerConfig.Route.Provenance = definitions.Provenance(storedProvenance)
```

Comment:
Let's just ignore this field at all. This is an artifact from old days when we had to patch the model with provenance status to return via blob API